conf/messages

```diff
@@ -593,7 +593,7 @@ hts.help-information.section.payments.s3.p4=If there is a problem, check with yo
 
 # Help/information: Bonuses
 hts.help-information.section.bonuses.title=Bonuses
-hts.help-information.section.bonuses.s1.title=What bonuses you can get (video)
+hts.help-information.section.bonuses.s1.title=What bonuses you can get
 hts.help-information.section.bonuses.s1.p1=You can earn 2 tax-free bonuses.
```

Comment:
I think you need to remove these two entries from messages.cy file too.

Reply:
Thanks, I have removed these now